kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/internal/parquet/ParquetFileReaderSuite.scala

```diff
@@ -15,58 +15,35 @@
  */
 package io.delta.kernel.defaults.internal.parquet
 
-import java.io.File
-import java.math.{BigDecimal, RoundingMode}
+import java.math.BigDecimal
 
-import scala.collection.mutable.ArrayBuffer
-
-import io.delta.golden.GoldenTableUtils.{goldenTableFile, goldenTablePath}
-import io.delta.kernel.defaults.internal.DefaultKernelUtils.DateTimeConstants
+import io.delta.golden.GoldenTableUtils.goldenTableFile
 import io.delta.kernel.defaults.utils.{ExpressionTestUtils, TestRow, VectorTestUtils}
 import io.delta.kernel.types._
 import org.scalatest.funsuite.AnyFunSuite
 
 class ParquetFileReaderSuite extends AnyFunSuite
   with ParquetSuiteBase with VectorTestUtils with ExpressionTestUtils {
 
-  def getSingleParquetFile(directory: File): String = {
-    val parquetFiles = directory.listFiles().filter(_.getName.endsWith(".parquet"))
-    assert(parquetFiles.size == 1)
-    parquetFiles.head.getAbsolutePath
-  }
-
-  //////////////////////////////////////////////////////////////////////////////////
-  // Decimal type tests
-  //////////////////////////////////////////////////////////////////////////////////
-
-  private val DECIMAL_TYPES_DICT_FILE_V1 = getSingleParquetFile(
-    goldenTableFile("parquet-decimal-dictionaries-v1"))
-
-  private val DECIMAL_TYPES_DICT_FILE_V2 = getSingleParquetFile(
-    goldenTableFile("parquet-decimal-dictionaries-v2"))
-
   test("decimals encoded using dictionary encoding ") {
+    val DECIMAL_DICT_FILE_V1 = goldenTableFile("parquet-decimal-dictionaries-v1").getAbsolutePath
```

Comment:
nit: can you convert the variable name to camelCase: `decimalDictFileV1`? Same for other other ones.